vm/2.0/modules/ROOT/pages/vm-examples.adoc

```diff
@@ -0,0 +1,20 @@
+= VM Connector Examples - Mule 4
+:page-aliases: connectors::vm/vm-connector-examples.adoc
+
+To fully benefit from using Anypoint Connector for Virtual Machine (VM) (VM Connector), see the following examples of configuring the connector operations:
+
+* xref:vm-dynamic-routing.adoc[Dynamic Routing with the VM Connector] +
```

Comment:
Ah! I now see that it is more likely that the peer topic titles should not use the imperative, because they do not include steps. So...Dynamic Routing..., Propagating..., Sending.... for the containers of imperative-titled tasks.

Reply:
I hope I haven't confused you!
TASK = STEPS = <DO THIS> title
Container topic with task subtopics = super-task, without its own steps, or mixed container topic of subtasks and concepts and reference, but no steps itself.